### PR TITLE
Web Forms

### PR DESCRIPTION
**Pull request by adrian2404**
added config module
discovering WTForms
added login form
using url_for instead of direct links

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 from flask import Flask
+from config import Config
```
Comment (RomanDerkach) [inline app/__init__.py]: pep8 says that there should be a space between libs and your own package.

File: app/routes.py [hunk 2]
```diff
@@ -18,3 +19,13 @@ def index():
...
     ]
     return render_template('index.html', title='Home', user=user, posts=posts)
+
+
+@app.route('/login', methods=['GET', 'POST'])
+def login():
+    form = LoginForm()
+    if form.validate_on_submit():
+        flash('Login requested for user {}, remember_me={}'.format(form.username,
+                                                                   form.remember_me))
```
Comment (RomanDerkach) [inline app/routes.py]: you probably forgot to get data attribute of the form. 
`form.username.data`

File: config.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import os
+
+
+class Config(object):
+    SECRET_KEY = os.environ.get('SECRET_KEY') or 'secret-key'
```
Comment (RomanDerkach) [inline config.py]: try to come up with more interesting `secret-key` :p






just kidding

**Review by RomanDerkach — CHANGES_REQUESTED** [review]

File: app/__init__.py (hunk 1 above)
Comment (adrian2404) [reply app/__init__.py]: my pep8 plugin in PyCharm didn't mention this, but I will add it.

Comment (RomanDerkach) [reply app/__init__.py]: от блін треба і собі поставити pep8 formatter

**Review by RomanDerkach — APPROVED** [review]
:1st_place_medal:

File: app/__init__.py (hunk 1 above)
Comment (adrian2404) [reply app/__init__.py]: можемо попробувати юзати tox

**Pull request merged**